infrastructure/main.tf

```diff
@@ -31,12 +31,12 @@ data "azurerm_key_vault_secret" "s2s_url" {
 }
 
 data "azurerm_key_vault_secret" "s2s_secret" {
-  name = "microservicekey-rd-fee-pay-ref-data-load"
+  name = "microservicekey-rd-location-ref-data-load"
   key_vault_id = "${data.azurerm_key_vault.s2s_key_vault.id}"
```

Comment:
same we don't need  s2s-secret as it is scheduling application.Please refer https://github.com/hmcts/rd-judicial-data-load